source/VMelnalksnis.PaperlessDotNet/Documents/DocumentContent.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 Valters Melnalksnis
+// Licensed under the Apache License 2.0.
+// See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace VMelnalksnis.PaperlessDotNet.Documents;
+
+/// <summary>
+/// The content of a document.
+/// </summary>
+public sealed class DocumentContent
+{
+	/// <summary>Initializes a new instance of the <see cref="DocumentContent"/> class.</summary>
+	/// <param name="content">The document content.</param>
+	/// <param name="fileName">The name of the file.</param>
+	/// <param name="contentType">The content-type of the file.</param>
+	public DocumentContent(Stream content, string fileName, string contentType)
+	{
+		Content = content;
+		FileName = fileName;
+		ContentType = contentType;
+	}
+
+	/// <summary>
+	/// Gets the document content.
+	/// </summary>
+	public Stream Content { get; init; }
```

Comment:
No need to have `init` on properties if they must be set from constructor
```suggestion
	public Stream Content { get; }
```